File: CS/WinForms_Localizer/WinForms_Localizer/MyDashboardWinLocalizer.cs
using DevExpress.DashboardWin.Localization;
using DevExpress.Utils.Localization;
using System;

namespace WinForms_Localizer
{
    internal class MyDashboardWinLocalizer : DashboardWinLocalizer
    {
        public override string GetLocalizedString(DashboardWinStringId id)
        {
            return Enum.GetName(typeof(DashboardWinStringId), id);
        }
    }
}

File: VB/WinForms_Localizer/WinForms_Localizer/MyDashboardWinLocalizer.vb
Imports DevExpress.DashboardWin.Localization
Imports System

Namespace WinForms_Localizer

    Friend Class MyDashboardWinLocalizer
        Inherits DashboardWinLocalizer

        Public Overrides Function GetLocalizedString(ByVal id As DashboardWinStringId) As String
            Return [Enum].GetName(GetType(DashboardWinStringId), id)
        End Function
    End Class
End Namespace
